diff --git a/fz.sh b/fz.sh
index 4fc1ab8..856baf1 100644
--- a/fz.sh
+++ b/fz.sh
@@ -278,15 +278,15 @@ __fz_init_zsh_completion() {
 
 _fz() {
   local rc
-  if [[ "$($FZ_HISTORY_LIST_GENERATOR "$@" | head | wc -l)" -gt 0 ]]; then
-    "$FZ_HISTORY_CD_CMD" "$@"
+  local err
+  local zmatch="$($FZ_HISTORY_LIST_GENERATOR "${@%/}" | head -n1)"
+  if [[ ${#zmatch} -gt 0 ]]; then
+    "$FZ_HISTORY_CD_CMD" "${@%/}"
   elif [[ "$FZ_SUBDIR_TRAVERSAL" -ne 0 ]]; then
-    err=$(builtin cd "${@: -1}" 2>&1)
-    rc=$?
-    if ! builtin cd "${@: -1}" 2>/dev/null; then
-      echo ${err#* } >&2
-      return $rc
-    fi
+    zmatch="$($FZ_HISTORY_LIST_GENERATOR "${@[0,-2]}" | head -n1)"
+    (){ builtin cd "$zmatch/"*"${@: -1}"*"/" 2> $1; rc=$? err=$(<$1);} =(:) "${@}"
+    echo ${err#* } >&2
+    return $rc
   fi
 }
 
