Support zsh-edit
diff --git a/autopair.zsh b/autopair.zsh
index 5477387..43c3299 100644
--- a/autopair.zsh
+++ b/autopair.zsh
@@ -4,7 +4,7 @@ AUTOPAIR_INHIBIT_INIT=${AUTOPAIR_INHIBIT_INIT:-}
 AUTOPAIR_BETWEEN_WHITESPACE=${AUTOPAIR_BETWEEN_WHITESPACE:-}
 AUTOPAIR_SPC_WIDGET=${AUTOPAIR_SPC_WIDGET:-"$(bindkey " " | cut -c5-)"}
 AUTOPAIR_BKSPC_WIDGET=${AUTOPAIR_BKSPC_WIDGET:-"$(bindkey "^?" | cut -c6-)"}
-AUTOPAIR_DELWORD_WIDGET=${AUTOPAIR_DELWORD_WIDGET:-"$(bindkey "^w" | cut -c6-)"}
+AUTOPAIR_DELWORD_WIDGET=backward-kill-subword
 
 typeset -gA AUTOPAIR_PAIRS
 AUTOPAIR_PAIRS=('`' '`' "'" "'" '"' '"' '{' '}' '[' ']' '(' ')' ' ' ' ')
@@ -191,7 +191,7 @@ autopair-delete() {
     zle ${AUTOPAIR_BKSPC_WIDGET:-backward-delete-char}
 }
 
-autopair-delete-word() {
+autopair-backward-kill-word() {
     _ap-can-delete-p && RBUFFER=${RBUFFER:1}
     zle ${AUTOPAIR_DELWORD_WIDGET:-backward-delete-word}
 }
@@ -203,7 +203,7 @@ autopair-init() {
     zle -N autopair-insert
     zle -N autopair-close
     zle -N autopair-delete
-    zle -N autopair-delete-word
+    zle -N autopair-backward-kill-word
 
     local p
     for p in ${(@k)AUTOPAIR_PAIRS}; do
@@ -218,8 +218,7 @@ autopair-init() {
     done
 
     bindkey "^?" autopair-delete
-    bindkey "^h" autopair-delete
-    bindkey "^w" autopair-delete-word
+    bindkey "^H" autopair-backward-kill-word
     bindkey -M isearch "^?" backward-delete-char
     bindkey -M isearch "^h" backward-delete-char
     bindkey -M isearch "^w" backward-delete-word
