--- marlonrichert---zsh-edit/zsh-edit.plugin.zsh	2021-08-26 17:52:53.358880585 -0400
+++ marlonrichert---zsh-edit/zsh-edit.plugin.zsh-patch	2021-08-26 17:52:50.040887434 -0400
@@ -10,7 +10,7 @@
   local dir=${${(%):-%x}:A:h}
   local fdir=$dir/functions
   typeset -gU FPATH fpath=( $dir $fpath )
-  autoload -Uz $fdir/bind $fdir/[._]edit.*~*.zwc(DN)
+  autoload -Uz $fdir/bind(.:t) $fdir/[._]edit.*~*.zwc(DN.:t)
 
   .beginning-of-buffer() {
     CURSOR=0
