--- marlonrichert---zsh-edit/zsh-edit.plugin.zsh	2021-04-18 16:40:07.191110037 -0400
+++ marlonrichert---zsh-edit/zsh-edit.plugin.zsh-patch	2021-04-18 16:44:04.806847115 -0400
@@ -6,13 +6,13 @@
   setopt $_edit_opts
 
   local fdir=${${(%):-%x}:h}/functions
-  unfunction bindkey 2>/dev/null
-  autoload -Uz $fdir/bindkey $fdir/_*~*.zwc
+  autoload -Uz $fdir/_edit_subword
 
   local widget
   for widget in {forward,backward,kill,backward-kill}-word; do
     zle -N $widget _edit_subword
   done
+  return
   for widget in yank yank-pop reverse-yank-pop; do
     zle -N $widget _visual_yank
   done
