--- marlonrichert---zsh-edit/zsh-edit.plugin.zsh	2021-04-01 17:29:32.791224046 -0400
+++ marlonrichert---zsh-edit/zsh-edit.plugin.zsh-patch	2021-04-01 17:31:30.785943564 -0400
@@ -6,12 +6,13 @@
   setopt $_edit_opts
 
   local fdir=${${(%):-%x}:h}/functions
-  autoload -Uz $fdir/bindkey $fdir/_*~*.zwc
+  autoload -Uz $fdir/_edit_subword
 
   local widget
   for widget in {forward,backward,kill,backward-kill}-word; do
     zle -N $widget _edit_subword
   done
+  return
   for widget in yank yank-pop reverse-yank-pop; do
     zle -N $widget _visual_yank
   done
