--- zdharma---history-search-multi-word/history-search-multi-word	2020-08-23 06:19:40.538991017 -0400
+++ zdharma---history-search-multi-word/history-search-multi-word-patch	2020-08-23 06:19:35.409006283 -0400
@@ -19,6 +19,8 @@
 # zstyle ":plugin:history-search-multi-word" synhl "yes"
 # zstyle ":plugin:history-search-multi-word" clear-on-cancel "no"
 
+[[ -z $HISTORY_BASE ]] && HISTORY_BASE="$HOME/.directory_history"
+[[ -z $PER_DIRECTORY_HISTORY_TOGGLE ]] && PER_DIRECTORY_HISTORY_TOGGLE='^G'
 [[ "$__HSMW_MH_SOURCED" != "1" ]] && source "$HSMW_REPO_DIR/hsmw-highlight"
 local right_brace_is_recognised_everywhere
 integer path_dirs_was_set multi_func_def ointeractive_comments
@@ -169,7 +171,11 @@
         # Colorify
         #
 
-        local preamble=$'\n'"Ctrl-K – context, Ctrl-J – bump. Entry #$__hsmw_hcw_index of $max_index"$'\n'
+        local history_type=""
+        if (( ${+_per_directory_history_is_global} )); then
+          history_type=", Ctrl-G – $(if [[ $_per_directory_history_is_global == false ]]; then echo 'global'; else echo "local"; fi)"
+        fi
+        local preamble=$'\n'"Ctrl-K – context, Ctrl-J – bump$history_type. Entry #$__hsmw_hcw_index of $max_index"$'\n'
         integer offset=${#preamble}+${#BUFFER}
         POSTDISPLAY="$preamble$text"
 
