--- customizepkg	2020-05-10 17:05:58.775716458 -0400
+++ customizepkg-patch	2020-05-10 17:05:26.950330824 -0400
@@ -18,7 +18,7 @@
 
 NAME=$(basename $0)
 CONFIGDIR="/etc/${NAME}.d"
-LOCALCONFIGDIR=${CUSTOMIZEPKG_CONFIG:-"${HOME}/.${NAME}"}
+LOCALCONFIGDIR=${CUSTOMIZEPKG_CONFIG:-"${HOME}/.config/${NAME}"}
 
 # use colordiff if it's available
 [[ $(type -p colordiff) ]] && DIFFCMD="colordiff -ub" || DIFFCMD="diff -ub"
